README.md

```diff
@@ -1,5 +1,8 @@
-![readme-banner](https://github.com/DroidKaigi/conference-app-2023-images/assets/1386930/22efb1a7-5a43-469c-a493-fa619034863e)
-
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/136104152/264391682-ae2b1067-966f-47a2-aa10-4e1ee4d09418.png">
```

Comment:
The image of Dirk is
https://github.com/DroidKaigi/conference-app-2023-images/assets/
It is believed that they need to be placed under the distribution.

Reply:
@takahirom 
Can you please put the Dark image in the same place as the Light and give us the URL?🙇‍♂️

Reply:
The reason we stored the image in a separate repository is as follows:

* We were developing in a private repository.
* Images added while the repository is private remain invisible even after making the repository open source.


You can now add images to the conference-app-2023 repository. Simply drag and drop the image into GitHub's web editor to do so.

Reply:
@takahirom 
I see!
Then it looks like we should keep the current references, so we will remove the Draft status of PR.